src/services/firestore.js

```diff
@@ -201,3 +201,12 @@ export const setUserIsMentor = (userId, eventId, isMentor) => {
       console.warn(`User not found ${userId}`);
     });
 };
+
+export const setDate = (eventId, period, date) => {
+  return db
+    .collection("events")
+    .doc(eventId)
+    .get()
+    .then((querySnapshot) => querySnapshot.docs)
+    .then((event) => event.data()[period] === date);
+};
```

Comment:
@andtos90 _help wanted_ on this 😅I thought this function could take:
* `eventId`
* `period` (e.g. `startDate` or `endDate`, these are the fields created on the DB)
* `date`, the new value to store